modules/openy_gc_demo/content/file/581596f7-2afa-4be1-9662-45d4c2772d60.json

```diff
@@ -0,0 +1,95 @@
+{
+    "_links": {
+        "self": {
+            "href": "http:\/\/rose.openy.docksal\/sites\/rose\/files\/video_thumbnails\/405566465.jpg"
```

Comment:
seems like unreliable demo content url

Reply:
This was generated by **default_content** module, on builds with the different domain, it works fine. Anyway, this was a temp solution, I think in the next phase we will switch to something better